.buildkite/scripts/release/k8s-manifests.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License 2.0;
+# you may not use this file except in compliance with the Elastic License 2.0.
+
+# Script to upload ECK k8s manifests to S3.
+
+set -eu
+
+ROOT="$(cd "$(dirname "$0")"; pwd)/../../.."
+# shellcheck disable=SC1091
+source "$ROOT/.env"
+
+if [[ "$IMG_VERSION" == "" ]]; then
```

Comment:
Correct me here please (bash is not my strongest), but isn't `-Z` usually used for this operation?

Reply:
Yes, we can do the same with `-z $IMG_VERSION` but I got used to using `== ""` which is a bit more readable.

Reply:
I can live with that. I certainly understand what `== ""` is by just looking at it, agreed.